commands/command.md

```diff
@@ -1,52 +1,79 @@
 Returns @array-reply of details about all Redis commands.
 
-Cluster clients must be aware of key positions in commands so commands can go to matching instances,
-but Redis commands vary between accepting one key,
-multiple keys, or even multiple keys separated by other data.
+Cluster clients must be aware of key positions in commands so commands can go to matching instances, but Redis commands vary between accepting one key, multiple keys, or even multiple keys separated by other data.
+
+Starting from Redis 7.0.0 a more flexible mechanism was introduced in order to help clients finding the key positions.
+For more information please check the [key-specs page][tr].
+[tr]: /topics/key-specs
 
 You can use `COMMAND` to cache a mapping between commands and key positions for
 each command to enable exact routing of commands to cluster instances.
 
-## Nested Result Array
-Each top-level result contains six nested results.  Each nested result is:
-
-  - command name
-  - command arity specification
-  - nested @array-reply of command flags
-  - position of first key in argument list
-  - position of last key in argument list
-  - step count for locating repeating keys
+`COMMAND` has several subcommands:
+
+ - `COMMAND INFO`
+ - `COMMAND COUNT`
+ - `COMMAND GETKEYS`
+ - `COMMAND LIST`
+ - `COMMAND DETAILS`
+
+`COMMAND`'s reply is an @array-reply, where each element is an @array-reply by itself, containing the following elements:
+
+ - command name
+ - command arity specification
+ - nested @array-reply of command flags
+ - position of first key in argument list
+ - position of last key in argument list
+ - step count for locating repeating keys
+ - nested @array-reply of [ACL categories][ta]
+ - nested @array-reply of [command tips][tb]
+ - nested @array-reply of [key-specs][td]
+ - nested @array-reply of subcommands
+[ta]: /topics/acl
+[tb]: /topics/command-tips
+[td]: /topics/key-specs
```

Comment:
no sure about this markdown, is it sane?

Reply:
@itamarhaber ?

Reply:
The syntax is ok, but there should be a \n before these just for safety (i.e. regular md is ok with that, but the current site's implementation is more sensitive).

Reply:
@itamarhaber what do you mean? an empty line between the `- nested` list and the `[ta]` thing?
i must admit i don't understand any of this.. what does it do?

anyway, and you add a suggestion to fix it, or even just push a fix?

Reply:
Yes, will push the fixes.